src/Tools/Extensions.ApiDescription.Server/src/build/Microsoft.Extensions.ApiDescription.Server.targets

```
@@ -63,6 +63,10 @@
       <_DotNetGetDocumentCommand>$(_DotNetGetDocumentCommand) $(OpenApiGenerateDocumentsOptions)</_DotNetGetDocumentCommand>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <OpenApiGenerateDocumentsOptions Condition=" '$(OpenApiGenerateEnvironment)' != '' " > $(OpenApiGenerateDocumentsOptions) --environment &quot;$(OpenApiGenerateEnvironment)&quot;</OpenApiGenerateDocumentsOptions>
```

I don't think you want to be adding an extra space here.

```suggestion
      <OpenApiGenerateDocumentsOptions Condition=" '$(OpenApiGenerateEnvironment)' != '' " >$(OpenApiGenerateDocumentsOptions) --environment &quot;$(OpenApiGenerateEnvironment)&quot;</OpenApiGenerateDocumentsOptions>
```